### DISCUSSION TITLE
ToonSquid Community Monthly Challenge: November

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#684] [Show and tell]
I decided that I would post the ToonSquid Challenge here for those who don’t use social media or is not Following the ToonSquid Community:

It’s time for the ToonSquid Community Monthly Challenge!

The theme of challenge this month is ‘Special Run’.
Due at November 30th. 
Submit your entry with the tag: #toonsquidchallenge (on Instagram or X/Twitter)
or just reply it down below

(Watch video below for more infomation)

https://github.com/user-attachments/assets/d2cc45d5-5a52-4fe1-b60c-054e12218981

**Comment by Gottafindaname** [comment]
There are too many challenges going on….

there are like three or sumn???

**Reply by Lioncub901** [reply]
True, I tried to merge with the discord one, but the Admin stated it would probably be best to make my own one so I decided to. Thanks for the feedback though.

**Comment by ZafuMusic** [comment]
Dude are you the guy harassing others in the comments? I'm out then

**Reply by Lioncub901** [reply]
That is fine. You could join the discord’s challenge. Just have fun animating in ToonSquid. Also I heard you are new to ToonSquid. Welcome, hope you enjoy your time using the app.

**Reply by Rohitgit2412** [reply]
I think 'harassing others' is a bit misleading here. Having an argument with someone who has a different opinion isn't the same as harassing others, which make it sounds like he is causing trouble for a lot of people who aren’t directly involved in the matter.

**Comment by DWrightAnimation** [comment]
Here's my entry: [My Entry](https://x.com/wrightimation/status/1858904496476176747?s=61&t=QQyooyNOMaYSA1slJ4aD2g)

**Comment by DWrightAnimation** [comment]
When will the December monthly challenge be announced?

**Reply by Lioncub901** [reply]
it should be out by tomorrow. It won’t have a animation but just the theme. Thanks.

**Reply by DWrightAnimation** [reply]
> it should be out by tomorrow. It won’t have a animation but just the theme. Thanks.

I've been eyeing the Twitter page for the theme, but the post isn't live yet. I don't mind if the theme doesn't have an example animation, so long as the theme can get my creative juices flowing.

**Comment by DWrightAnimation** [comment]
Here's my entry for December 2024: [My Entry](https://x.com/wrightimation/status/1870948528354464192?s=61&t=QQyooyNOMaYSA1slJ4aD2g)

**Comment by DWrightAnimation** [comment]
When will January's challenge be announced? The later these challenges are announced, the less time we have to work on them.

**Reply by Lioncub901** [reply]
Sorry for being late: check the twitter page for the theme. Been working on some projects in the background. Some to do with the web.

**Reply by Lioncub901** [reply]
also thanks for showing interest

**Comment by DWrightAnimation** [comment]
It's already April 16th, and no update on this month's challenge theme.

**Reply by Lioncub901** [reply]
I apologize I just been busy with college. There will be challenge on May (I want to make an animation for that (I hope)), sorry about that.

**Reply by DWrightAnimation** [reply]
To fill in the space I have for April, could you give me an idea of what I could put in April's place?

**Reply by Lioncub901** [reply]
how about gardening.

**Reply by DWrightAnimation** [reply]
Sounds good to me.

**Reply by DWrightAnimation** [reply]
I haven't finished April's animation yet, but what will the theme for May be? Just asking so I have time to prepare.

**Comment by DWrightAnimation** [comment]
May 2025 Theme?
